### ISSUE TITLE
[9.5] - Missing information in some tickets when creating with mailgate

### ISSUE THREAD

**Issue by andrepetinga** [glpi-project/glpi#8383]
**Describe the bug**

We have noticied that some information is missing in some tickets, when created from mailgate.
We don't know why this is happening.

Here is an example. This is the end of the created ticket

![image](https://user-images.githubusercontent.com/43515938/101157060-70d8a900-3621-11eb-8416-2bcd4ca103b9.png)

And this is the end of the email present in the server

![image](https://user-images.githubusercontent.com/43515938/101157297-cdd45f00-3621-11eb-91ff-b10884df727c.png)

Notice that the text "jumps" from the signature of one person's email to the previous one

![image](https://user-images.githubusercontent.com/43515938/101157820-a29e3f80-3622-11eb-8bc4-dcfc39624085.png)

**Expected behavior**

Ticket should be created with all email information.

If you want, I can forward the email to your system configuration, so that we can check if it's created the same way.

**Comment by OscarBeiro** [comment]
Olá André,

GLPI sanitizes emails before ticket been created. This is a complex subject as every email client has its own quirks.
For instance: Signatures (not always as you see) and especially on email replies, quoted text like the one on your example.

So, some elements are been removed, this is the default behaviour and I think it is OK. 

Regards,

**Comment by andrepetinga** [comment]
Yes, I understand there are several email clients... But this cases are complicated, because we may lose important information for the ticket. Of course if it says something like "according to the info below" and there is nothing below, we know something went wrong. But this is not always the case...
And we have many, many emails similar to this one, so I tried to understand what was the difference, but with no luck...

**Comment by tomolimo** [comment]
Hello @OscarBeiro 
I totally agree with @andrepetinga: GLPI sanitize process must not delete content!
Thank you,
Regards,
Tomolimo

**Comment by FrankGraux** [comment]
We're experiencing the same issues with some e-mails. This is very annoying since now we cannot rely on the tickets to be complete.

**Comment by RodolpheLP** [comment]
> Hello @OscarBeiro
> I totally agree with @andrepetinga: GLPI sanitize process must not delete content!
> Thank you,
> Regards,
> Tomolimo

Hello,
I'm agree with that you said : GLPI sanitize process must not delete content!
Some important information is missing regulary from mails integtrated by GLPI and technicians  missed useful elements by this way.

Thanks if you could undone that, it'll be usefull.

Regards

**Comment by patrickpcs91** [comment]
Hello there! 

Under no circumstance mail content should be deleted or truncated.
I’ve upgraded from 9.4.5 to 9.5.3 and all seemed fine. Only a few weeks later down the road we noticed this issue because of costumers and technicians getting upset with the missleading info in the ticket.
I guess this is quite an issue. Anyone who relies on mailgate should not use version 9.5.2 or higher (I tested from 9.4.5 to 9.5.5). I’m amazed that there isn’t many complains about this issue here, maybe many others will only notice it when a customer reaches out.
I want to provide a mail for testing, but it’s gona take time to get rid of all sensitive info in it.

This is how I’m working around this for now….
We've turned off the mail gate and all tickets are been created manually by our N1 staff.
Since it’s hard to spot this issue in a timely manner and we’ve a considerable volume of tickets daily, backup doesn’t matter. I cannot just get rid of thousands of tickets =/
I’m trying to downgrade back to version 9.5.1 as there wasn’t a lot of changes in the database schema according with the migration log.
_Migration to 9.5.2
glpi_appliancerelations (We have no management of appliances on glpi)
glpi_mailcollectors (I’m worried but it’s working on test environment)
glpi_appliances (We have no management of appliances on glpi)
glpi_states (We do not use the “ASSETS” module)
Migration to 9.5.3
Nothing_
Just use the 9.5.1 app version and alter in the database glpi_config version and dbversion to 9.5.0. Yep, 9.5.ZERO.
